ares/fc/controller/sfcgamepad/sfcgamepad.cpp

```diff
@@ -0,0 +1,94 @@
+SFC_Gamepad::SFC_Gamepad(Node::Port parent) {
```

Comment:
nit: We don't usually use underscores in class names; consider `SuperFamicomController`, as SFCController looks awkward.

Reply:
Make whatever changes you want, dont have time to deal with this right now, personal things are happening in my life and its likey 5 seconds for you anyways....

Same with the other pull request for those Coleco-Vision Mappers.